raiden-ts/tests/integration/integration.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { RaidenState } from 'raiden-ts/state';
+import { assert, Storage } from 'raiden-ts/utils/types';
+import { Raiden } from 'raiden-ts/raiden';
+import fs from 'fs';
+import { ContractsInfo } from 'raiden-ts/types';
+import { bigNumberify, parseEther } from 'ethers/utils';
+import { MockStorage } from '../e2e/mocks';
+import { filter } from 'rxjs/operators';
+import { Signer, Wallet } from 'ethers';
+
+jest.setTimeout(80000);
+
+const signer = new Wallet('0x0123456789012345678901234567890123456789012345678901234567890123');
+//const partner1 = '0x517aAD51D0e9BbeF3c64803F86b3B9136641D9ec';
+const partner2 = '0xCBC49ec22c93DB69c78348C90cd03A323267db86';
+
+async function createRaiden(
+  account: number | string | Signer,
+  stateOrStorage?: RaidenState | Storage,
+): Promise<Raiden> {
+  const deploymentInfoFile = process.env.DEPLOYMENT_INFO;
+  const deploymentServiceInfoFile = process.env.DEPLOYMENT_SERVICES_INFO;
+
+  assert(deploymentInfoFile !== undefined);
+  assert(deploymentServiceInfoFile !== undefined);
+  assert(fs.existsSync(deploymentInfoFile));
+  assert(fs.existsSync(deploymentServiceInfoFile));
+
+  const options = { encoding: 'utf8', flag: 'r' };
+
+  const deployFile = await fs.readFileSync(deploymentInfoFile, options);
```

Comment:
`*Sync` methods don't need `await`, they're synchronous, but are also blocking. Prefer [fs.promises](https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_promises_api) APIs (async)

Reply:
I was trying to find the promises api but I could only find the callback one and I probably missed the await there, thanks for noticing :)